### DISCUSSION TITLE
Font throwing an error when run from EXE (Auto Py to EXE)

### DISCUSSION THREAD

**Discussion by justinmbaron** [TomSchimansky/CustomTkinter#1153] [Q&A]
Hi

Firstly, here is the code which runs fine in PyCharm with v5.0..3 of customtkinter

`def display_button(button_text):
    submit_button = customtkinter.CTkButton(root,
                                            text=button_text,
                                            border_width=2,
                                            corner_radius=8,
                                            width=120,
                                            height=32,
                                            font=("Arial", 20),
                                            command=main_process)
    submit_button.pack(pady=20, padx=10)`

When complied (with additional files pointing to the customtkinter folder in site-packages) I get the following error on execution:

  File "customtkinter\widgets\ctk_label.py", line 54, in __init__
TypeError: type object got multiple values for keyword argument 'font'
[15188] Failed to execute script Recall using tags v2

If I comment out the font line all works OK.

Any thoughts - it is possible I have multiple versions of the library on my PC but I am pretty sure I am pointing to the correct one?

Thanks!
Justin Baron

**Comment by VollGerne** [comment]
Have you tried to youse CTKFont instead directly assign a font to the widget?
https://github.com/TomSchimansky/CustomTkinter/wiki/CTkFont

**Reply by justinmbaron** [reply]
Thank you s much for responding.  I did try that with worse results - HOWEVER, I have solved the issue by deleting the old version of customtkinter.
I had to copy the v5 customtkinter folder to C:\Users\Justin Baron\AppData\Roaming\Python\Python38\site-packages as Auto Py to EXE didn't seem to pick up the folder in my actual project site-package where it was originally installed even though I pointed to it.
All good now!
